js/main.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+const api_URL = 'https://opentdb.com/api.php?amount=10&type=multiple';
+const questionNum = document.getElementById('questionNum');
+const question = document.getElementById('question');
+const choices = document.getElementById('choices');
+const btn = document.getElementById('btn');
+const category = document.getElementById('category');
+const difficulty = document.getElementById('difficulty');
+let quizSet = [];
+let currentNum = 0;
+let score = 0;
+// クイズ取得
+const fetchQuizData = () => {
+  question.textContent = '少々お待ちください';
+  questionNum.textContent = '取得中';
+  btn.hidden = true;
+  fetch(api_URL)
+  .then(response => response.json())
+  .then((data) => {
+    quizSet.push(...data.results);
+    setQuiz();
```

Comment:
`await` を使うとこんな感じになります。

```suggestion
  const response = await fetch(api_URL);
  const data = await response.json();
  quizSet.push(...data.results);
  setQuiz();
```